### PR TITLE
Associate report to node

### PR DESCRIPTION
**Pull request by FranckLecuyer**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
Import case reports are associated to the network uuid

**What is the new behavior (if this is a feature change)?**
Import case reports are now associated to a specific report uuid

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by laurgarn — APPROVED** [review]
Nothing bad detected

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java [hunk 1]
```diff
@@ -125,38 +125,38 @@ private static EquipmentInfos toEquipmentInfos(Identifiable<?> i, UUID networkUu
             .build();
     }
 
-    NetworkInfos importCase(UUID caseUuid, String variantId) {
+    NetworkInfos importCase(UUID caseUuid, String variantId, UUID reportUuid) {
         CaseDataSourceClient dataSource = new CaseDataSourceClient(caseServerRest, caseUuid);
-        ReporterModel reporter = new ReporterModel("importNetwork", "import network");
+        ReporterModel reporter = new ReporterModel(UUID.randomUUID().toString(), "import network");
```
Comment (SlimaneAmar) [inline src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:130]: Root ?

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java [hunk 2]
```diff
@@ -288,7 +288,7 @@ private static CgmesExportContext createContext(Network mergedNetwork, List<Netw
 
     NetworkInfos importCgmesCase(UUID caseUuid, List<BoundaryInfos> boundaries) {
         if (CollectionUtils.isEmpty(boundaries)) {  // no boundaries given, standard import
-            return importCase(caseUuid, null);
+            return importCase(caseUuid, null, UUID.randomUUID());
```
Comment (SlimaneAmar) [inline src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:291]: Root ?

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java]: Not really important, but OK : changed

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:291]: No, here, an UUID is required

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**